spec.html

```diff
@@ -23722,8 +24631,8 @@ <h1>HostEnsureCanCompileStrings ( _callerRealm_, _calleeRealm_ )</h1>
       </emu-clause>
 
       <emu-clause id="sec-evaldeclarationinstantiation" aoid="EvalDeclarationInstantiation">
-        <h1>EvalDeclarationInstantiation ( _body_, _varEnv_, _lexEnv_, _strict_ )</h1>
-        <p>The abstract operation EvalDeclarationInstantiation takes arguments _body_, _varEnv_, _lexEnv_, and _strict_. It performs the following steps when called:</p>
+        <h1>EvalDeclarationInstantiation ( _body_, _varEnv_, _lexEnv_, _privateEnv_, _strict_ )</h1>
```

Comment:
It seems like there probably needs to be a change to this so that `new class { x = eval('arguments') }` is an error, as it is in engines.

It looks like maybe [this early error](https://tc39.es/proposal-class-fields/#sec-performeval-rules-in-initializer) got dropped? If you add it here, I note that the second bullet point seems redundant: class field initializers are specified by using actual functions, which means the existing logic in PerformEval for determining which rules to apply will work.